lib/index.js

```diff
@@ -87,23 +89,31 @@ function getOptions(config) {
     process.env.npm_config_hugo_bin_download_repo,
     config.downloadRepo
   ].find(Boolean) ?? 'https://github.com';
-  const isExtended = [
+  const rawBuildTags = [
     process.env.HUGO_BIN_BUILD_TAGS,
     process.env.npm_config_hugo_bin_build_tags,
     config.buildTags
-  ].includes('extended');
+  ].find(v => v !== null && v !== '') ?? '';
```

Comment:
isn't this the same as doing `filter(Boolean)`?

Reply:
yea very good point, i've switched this to filter